base/settings/R/check.all.settings.R

```diff
@@ -67,12 +67,17 @@ check.inputs <- function(settings) {
         }
       } else if ("path" %in% names(settings$run$inputs[[tag]])) {
         # can we find the file so we can set the tag.id
-        id <- PEcAn.DB::dbfile.id(
-          "Input",
-          settings$run$inputs[[tag]][["path"]],
-          dbcon,
-          hostname)
-        if (!is.na(id)) {
+        #adding for to loop over ensemble member filepaths 
+        id <- list()
+        for (j in 1:length(settings$run$inputs[[tag]][["path"]])){
+          path <- settings$run$inputs[[tag]][["path"]]
+          id[j] <- PEcAn.DB::dbfile.id(
+            "Input",
+            path[[j]],
+            dbcon,
+            hostname)
+        }
+        if (FALSE %in% is.na(id)) {
```

Comment:
Possibly clearer: 
```suggestion
        if (any(is.na(id))) {
```

Reply:
I like yours better I think it will be less error prone.